dashboards/couchdb/couchdb-prometheus-overview.json

```diff
@@ -0,0 +1,242 @@
+{
+  "category": "CUSTOM",
+  "displayName": "CouchDB Prometheus Overview",
+  "mosaicLayout": {
+    "columns": 12,
+    "tiles": [
+      {
+        "height": 4,
+        "widget": {
+          "title": "Requests By Method",
```

Comment:
Can you change the capitalization of "By" to "by" so it is consistent with the other chart titled "Responses by Status Codes"?